app/forms/conversion/involuntary/create_project_form.rb

```diff
@@ -9,7 +9,8 @@ def save
       provisional_conversion_date: provisional_conversion_date,
       advisory_board_date: advisory_board_date,
       regional_delivery_officer_id: user.id,
-      task_list: Conversion::Involuntary::TaskList.new
+      task_list: Conversion::Involuntary::TaskList.new,
+      assigned_to_regional_caseworker_team: false
```

Comment:
I don't think we need to implement anything for involuntary? We have the two subclasses for just this use case?

Reply:
Good point, I originally did this for both but then backtracked, and didn't backtrack enough 😆